### PR TITLE
Update navicat-data-modeler to 2.0.1

### PR DESCRIPTION
**Pull request by scribblemaniac**

### CODE REVIEW

File: Casks/navicat-data-modeler.rb [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 
   url "http://download.navicat.com/download/modeler0#{version.sub(%r{^(\d+)\.(\d+).*},'\1\2')}_en.dmg"
   name 'Navicat Data Modeler'
-  name 'Navicat Data Modeler'
   homepage 'http://www.navicat.com/products/navicat-data-modeler'
   license :commercial
   tags :vendor => 'Navicat'
 
+  depends_on :macos => '>= :lion'
+  depends_on :arch => :intel
```
Comment (vitorgalvao) [inline Casks/navicat-data-modeler.rb]: Same here, I doubt we have a meaningful (or any) amount of non-intel users.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as 8f303246855e015838ff686254eaead95d49854e to keep commit history cleaner. Your contribution is still credited to you.

File: Casks/navicat-data-modeler.rb (hunk 1 above)
Comment (scribblemaniac) [inline Casks/navicat-data-modeler.rb]: Should I not worry about adding depends_on for :intel or >= :tiger then, or are they good to have simply for completeness?

Comment (vitorgalvao) [inline Casks/navicat-data-modeler.rb]: They do no harm, but don’t worry about them. I’ll go even further and say you shouldn’t worry about anything slowe than Snow Leopard.

**Pull request closed without merging**